main.go

```diff
@@ -35,10 +35,13 @@ func main() {
 	silent := flag.Bool("silent", false, "Suppress output")
 	dryRun := flag.Bool("dry-run", false, "Consume input but do not send HTTP requests to targets")
 	timeout := flag.Int("timeout", 10, "HTTP client timeout in seconds")
+	connections := flag.Int("connections", 10000, "Max open idle connections per target host")
+	maxConnections := flag.Int("max-connections", 0, "Max connections per target host (default unlimited)")
 	strict := flag.Bool("strict", false, "Panic on bad input")
 	memprofile := flag.String("memprofile", "", "Write memory profile to `file` before exit")
 	cpuprofile := flag.String("cpuprofile", "", "Write cpu profile to `file` before exit")
-	numWorkers := flag.Int("workers", 1000, "Number of client workers to use")
+	numWorkers := flag.Int("workers", runtime.NumCPU()*2, "Number of client workers to use")
+	printStatsInterval := flag.Duration("print-stats", 0, "Prints statistics every X")
```

Comment:
@szmglh is this a flag, or should we use an environment variable to turn it on instead? If it's a flag, I think should make the text description more specific.

Reply:
I prefer to stay consistent and keep this as a flag. See new description in 470f8bcfef82a201fd49c67220af64df43b24b45